src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -13,8 +13,24 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <!-- Documentation warnings -->
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <TargetsForTfmSpecificBuildOutput>GatherProjectReferenceAssets</TargetsForTfmSpecificBuildOutput>
```

Comment:
`TargetsForTfmSpecificBuildOutput` will put the `Microsoft.Data.Analysis.Interactive` assembly in the `lib` folder. We explicitly don't want that. We need it to be in the `interactive-extensions/dotnet` folder in the nupkg.

Reply:
I see, I believe there was a misunderstanding here as @pgovind seemed to be under the impression that this is where we wanted it to live. In that case, same question as I asked above, how do interactive-extensions/dotnet usually ship? I assume carrying their own dependencies? 

We can still use TargetsForTfmSpecificBuildOutput and just set a different TargetPath or alternatively we could use TargetsForTfmSpecificContentInPackage instead.

Reply:
Yup, once @joperezr explained that having a wrapper nuget csproj wasn't a very robust way to packing 2 libs, I thought that left us with 2 options:
1. Ship MDAI and point notebook users to the MDAI package (while non-notebook users still use MDA). I don't think we should ever do this.
2. Ship MDAI with MDA, so the notebook users pick up the formatter automatically. Ideally we could've done this without modifying MDA.csproj, but this is where the robustness issue lies.

Just to be clear, I'm still open to having a wrapper nuget csproj if we decide that we absolutely don't want MDA to have a dependency on `Microsft.DotNet.Interactive`.

I'm just brain-storming here, but considering the nuget package issues(https://github.com/dotnet/interactive/tree/master/samples/extensions seems to be manually copying all the dlls during `pack`?), I wonder if there's another way for a library to say "If I'm loaded in an interactive environment, load these other nuget packages". Maybe some sort of asset(a text file?) in the library nuget package that lists other nuget packages to be downloaded by dotnet-interactive? (I don't know very much about package issues, so this could be a terrible idea!)

Reply:
What you are brainstorming seems to be similar to the convention that there seems to be in place already for stuff located under interactive-extensions/dotnet folder from @eerhardt's comment. So in this case, all we want is to keep stuff similar to what we have them now, except that instead of also adding MDAI into the lib/netcoreapp3.1 folder as we are doing now we should instead place it under the interactive-exteinsions folder and we should also remove the bit of adding the package references that are MDAI specific into the MDA package as from @eerhardt response this seems to be already present inbox on dotnet interactive, so no need to carry them with you nor have a package reference to them.